### PR TITLE
Add Stop Play/Recording Input Menu

### PR DESCRIPTION
**Pull request by RisingFog**
Allows the user to either stop playing a movie file on movie playback, or to stop recording a movie file on movie recording.

Resolves issue 9700: https://bugs.dolphin-emu.org/issues/9700

### CODE REVIEW

File: Source/Core/DolphinWX/MainMenuBar.cpp [hunk 1]
```diff
@@ -140,6 +140,7 @@ wxMenu* MainMenuBar::CreateMovieMenu() const
 
   movie_menu->Append(IDM_RECORD, _("Start Re&cording Input"));
   movie_menu->Append(IDM_PLAY_RECORD, _("P&lay Input Recording..."));
+  movie_menu->Append(IDM_STOP_RECORD, _("Stop Play/Recording Input"));
```
Comment (JosJuice) [inline Source/Core/DolphinWX/MainMenuBar.cpp]: Is it possible to make this change name at runtime? The current name feels a bit confusing to me. Also, please replace "Play" with "Playing" - that would probably also make it less confusing.

Comment (RisingFog) [inline Source/Core/DolphinWX/MainMenuBar.cpp]: Renamed it to Playing, but I'll see what I can do about renaming it depending on the action.

File: Source/Core/DolphinWX/FrameTools.cpp [hunk 2]
```diff
@@ -496,6 +497,30 @@ void CFrame::OnPlayRecording(wxCommandEvent& WXUNUSED(event))
     BootGame("");
 }
 
+void CFrame::OnStopRecording(wxCommandEvent& WXUNUSED(event))
+{
+  if (Movie::IsRecordingInput())
+  {
+    bool wasPaused = Core::GetState() == Core::State::Paused;
```
Comment (lioncash) [inline Source/Core/DolphinWX/FrameTools.cpp]: These can be `const bool`. Coding style also uses naming `like_this`.

Comment (RisingFog) [inline Source/Core/DolphinWX/FrameTools.cpp]: Done.

**Pull request merged**